src/main/java/com/meetup/server/place/presentation/PlaceController.java

```diff
@@ -35,4 +35,13 @@ public ApiResponse<?> confirmPlace(
     public ApiResponse<PlaceResponseList> getAllPlaces(@RequestParam UUID eventId) {
         return ApiResponse.success(placeService.getAllPlaces(eventId));
     }
+
+    @Operation(summary = "장소 상세 조회 API", description = "장소 ID를 통해 장소의 상세 정보를 조회합니다.")
+    @GetMapping("/{placeId}")
+    public ApiResponse<?> getPlaceDetails(
```

Comment:
```suggestion
    public ApiResponse<PlaceDetailResponse> getPlaceDetails(
```
반환 타입을 지정해주는게 TypeSafe할 거 같아요! 나중에 유지보수 하기에도 편할 거 같습니다